src/openforms/js/components/admin/RJSFWrapper.js

```diff
@@ -20,15 +21,20 @@ const CustomFieldTemplate = ({
     hidden,
     required,
     displayLabel,
+    schema,
 }) => {
+    const isNullBoolean = isSelect(schema) && schema.type.includes('boolean');
```

Comment:
How does this differentiate between a 'normal' bool and a nullable bool? I see that for the `attachFilesToEmail` field the type is `['boolean', 'null']`

Reply:
the `isSelect(schema)` looks at the presence of `enum` to determine if it's a select or a "regular" field. And our json-schema generator only outputs the enum if `field.allow_null is True`